scripts/SSRPlugin.ts

```diff
@@ -110,5 +84,12 @@ untilDocumentReady().then(() => {
         return regeneratedHTML
     }
 }
-
-module.exports = SSRPlugin
+function appendScript(html: string, src: string, position: string, position2: string) {
```

Comment:
Plz rename`postion` and `position2` to better name. Maybe `position => tagName` and `position2 => position`?